x/msgfees/wasm/encode.go

```diff
@@ -28,8 +28,10 @@ type AssessCustomFeeParams struct {
 	From string `json:"from"`
 	// An optional short name
 	Name string `json:"name,omitempty"`
-	// An optional address to receive the fees. if present, the total amount is sent to the address.
+	// An optional address to receive the fees. if present, the split amount from basis points is sent to address.
 	Recipient string `json:"recipient,omitempty"`
+	// An optional recipient basis points (0 - 10,000). if not present, defaults to 10,000
```

Comment:
same thing here as mentioned in my previous comments about using a constant/default param.

Reply:
See comment above.

Reply:
👍